boards.txt

```diff
@@ -365,6 +365,18 @@ generic.menu.sdk.nonosdk221=nonos-sdk 2.2.1 (legacy)
 generic.menu.sdk.nonosdk221.build.sdk=NONOSDK221
 generic.menu.sdk.nonosdk3v0=nonos-sdk pre-3 (180626 known issues)
 generic.menu.sdk.nonosdk3v0.build.sdk=NONOSDK3V0
+generic.menu.sdk.nonosdk300=nonos-sdk 3.0.0
+generic.menu.sdk.nonosdk300.build.sdk=NONOSDK300
+generic.menu.sdk.nonosdk301=nonos-sdk 3.0.1
+generic.menu.sdk.nonosdk301.build.sdk=NONOSDK301
+generic.menu.sdk.nonosdk302=nonos-sdk 3.0.2
+generic.menu.sdk.nonosdk302.build.sdk=NONOSDK302
+generic.menu.sdk.nonosdk303=nonos-sdk 3.0.3
+generic.menu.sdk.nonosdk303.build.sdk=NONOSDK303
+generic.menu.sdk.nonosdk304=nonos-sdk 3.0.4
+generic.menu.sdk.nonosdk304.build.sdk=NONOSDK304
+generic.menu.sdk.nonosdk305=nonos-sdk 3.0.5
+generic.menu.sdk.nonosdk305.build.sdk=NONOSDK305
```

Comment:
...do we actually need 3.0.0...3.0.4?

Reply:
Yeah, I am not sure all 3.0 are stable for everyone.